hasher-matcher-actioner/hmalib/common/models/bank.py

```diff
@@ -180,6 +186,8 @@ class BankMember(DynamoDBItem):
     is_removed: bool = field(default=False)
     is_media_unavailable: bool = field(default=False)
 
+    tags: t.Set[str] = field(default_factory=set)
```

Comment:
~nit: I was confused while reviewing if  bank_member tags were just a copy of the bank tags. 
I also had a slightly harder time differentiating the two as I traced through the logic of each.

Given that we call the id for this object ` bank_member_id` and the id for banks `bank_id` could be call the tags: `bank_member_tags` and `bank_tags` instead?

Reply:
I can get behind that. :) Thanks for flagging.